Index: javascript/Items.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/javascript/Items.js b/javascript/Items.js
--- a/javascript/Items.js	
+++ b/javascript/Items.js	
@@ -1,5 +1,5 @@
 const coffees = [
-    {imgSrc: "../images/Coffee/espressoSmall.png", mkTitle: "еспресо", enTitle: "espresso", price: 60},
+    {imgSrc: "../images/Coffee/kafe.png", mkTitle: "еспресо", enTitle: "espresso", price: 60},
     {imgSrc: "../images/Coffee/fredoEspresso.png", mkTitle:"фредо еспресо", enTitle: "fredo espresso", price: 70},
     {imgSrc: "../images/Coffee/macchiato.png", mkTitle:"макијато", enTitle: "macchiato", price: 70},
     {imgSrc: "../images/Coffee/shortMacchiato.png", mkTitle:"мало макијато", enTitle: "short macchiato", price: 60},
